### ISSUE TITLE
[Adaptive Cards - Samples - Activity update] - Narrator is not announcing any information about the date on navigating inside the calendar flyout in the 'Activity update sample' page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9306]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Screen Reader: Narrator, NVDA

**Repro Steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the above URL on the Edge Dev browser.
3. Navigate to the 'Samples' control present on the header section of the page using tab key and activate using enter key.
4. Navigate to the 'Activity Update' control present on the left navigation using tab key and activate using enter key.
5. Navigate to the 'Set due date' expandable control on the 'Adaptive Card' section using tab key and activate using enter key.
6. Navigate to the 'Calendar' icon using tab key and activate using enter key.
7. Now navigate inside the data using arrow keys.
8. Verify that, narrator is announcing any information about the date on navigating inside the calendar flyout in the 'Activity update sample' page or not.

**Actual Result:**
Narrator is not announcing any information about the date on navigating inside the calendar flyout in the 'Activity update sample' page. Currently narrator remains silent on navigating to the dates.

**Note**
Same issue does not replicate with NVDA.

**Expected Result:**
Narrator should announce the date information when the user navigates inside the calendar flyout on the 'Activity update sample' page. Each focused date should be read aloud clearly, including the day, month, and year. Narrator should also indicate the currently selected date. This ensures that screen reader users can understand and navigate the calendar effectively.

**User Experience:**
Visually impaired users who rely on screen reader to access the web content will be impacted if Narrator does not announce date information while navigating the calendar flyout. Users may be unable to identify the focused or selected date. This makes it difficult to navigate the calendar accurately. As a result, users may not be able to complete tasks that require date selection independently.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/26353e23-e548-454c-acd6-20fc5df02033

### Card JSON

```JSON
NA
```
